### PR TITLE
Sample should be limited

### PR DESCRIPTION
**Pull request by badboy**
The limit was recently increased. In order to record something into the
metrics ping, we now need to add more.

### Pull Request checklist ###

- [ ] **Quality**: This PR builds and tests run cleanly
  - `make test` runs without emitting any warnings
  - `make lint` runs without emitting any errors
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes a changelog entry or an explanation of why it does not need one
  - Any breaking changes to language binding APIs are noted explicitly

### CODE REVIEW

File: glean-core/examples/sample.rs [hunk 1]
```diff
@@ -64,14 +65,16 @@ fn main() {
     let ping = ping_maker
         .collect_string(glean.storage(), glean.get_ping_by_name("core").unwrap())
         .unwrap();
-    println!("Ping:\n{}", ping);
+    println!("Core Ping:\n{}", ping);
 
-    let mut long_string = std::iter::repeat('x').take(49).collect::<String>();
+    // Cause some error reporting, which will land in the metrics ping.
+    // The `local_metric` itself is not in the metrics ping.
+    let mut long_string = std::iter::repeat('x').take(99).collect::<String>();
```
Comment (Dexterp37) [inline glean-core/examples/sample.rs]: I think there was a test case for this already, why should we clutter this sample a bit more?

Comment (badboy) [reply glean-core/examples/sample.rs]: Hm, it was in there before and was useful for a quick one-off test.
But I'm also fine with keeping the sample much more minimal and thus removing this part.

Comment (Dexterp37) [reply glean-core/examples/sample.rs]: I personally feel like we should keep examples to the bare minimum :)

File: glean-core/examples/sample.rs [hunk 2]
```diff
@@ -19,59 +19,55 @@ fn main() {
...
     list.add(&glean, "once");
     list.add(&glean, "upon");
 
     let ping_maker = PingMaker::new();
     let ping = ping_maker
-        .collect_string(glean.storage(), glean.get_ping_by_name("core").unwrap())
+        .collect_string(glean.storage(), glean.get_ping_by_name("baseline").unwrap())
         .unwrap();
-    println!("Ping:\n{}", ping);
+    println!("Core Ping:\n{}", ping);
```
Comment (Dexterp37) [inline glean-core/examples/sample.rs]: nit: Baseline Ping?

**Pull request merged**